test/workerd/tests.mjs

```diff
@@ -0,0 +1,32 @@
+import assert from "node:assert";
+import process from "node:process";
+
+globalThis.process = process;
```

Comment:
Without this, workerd fail:

```
workerd/io/worker.c++:1950: info: uncaught exception; source = Uncaught (in promise); stack = ReferenceError: process is not defined
    at unenv/runtime/node/crypto:387:21
workerd/io/io-context.c++:348: info: uncaught exception; exception = workerd/jsg/_virtual_includes/jsg/workerd/jsg/value.h:1367: failed: jsg.ReferenceError: process is not defined
```

(process is used for `process.getBuiltinModule("node:crypto")`...)

@jasnell @anonrig is this expected?